tests/unit/meta/meta.ts

```diff
@@ -145,9 +140,9 @@ registerSuite({
 		const widget = new TestWidget();
 		widget.append(div);
 
-		resolveRAF();
-		resolveRAF();
-
-		assert.strictEqual(renders, 1, 'expected one renders');
+		assert.throws(() => {
+			resolveRAF();
```

Comment:
I think the first one performs an invalidate which is resolved in the second one.